server/api/order.js

```diff
@@ -0,0 +1,16 @@
+const orderRouter = require("express").Router();
+const { Order } = require("../db");
+
+// console.log("cheese", Cheese)
```

Comment:
Maybe get rid of these logs and then on line 6 it should get route for all orders